docs/userGuide/syntax/tree.md

```diff
@@ -0,0 +1,115 @@
+## Tree
+
+**A `tree` component allows you to generate tree-like visualisations, suitable for displaying folder structure.**
+
+Use indentation (2 spaces) to indicate the level of nesting.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+<tree>
+C:/course/
+  textbook/
+    index.md
+  index.md
+  reading.md
+  site.json
+</tree>
+</variable>
+</include>
+
+Markdown unordered lists are also supported.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+<tree>
+C:/course/
+  - textbook/
+  - index.md
+C:/course/
+  * textbook/
+  * index.md
+C:/course/
+  + textbook/
+  + index.md
+</tree>
+</variable>
+</include>
+
+<box type="info">
+
+Besides file system structures, you can use this for any lightweight tree-like structure.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+<tree>
+Enjoy MarkBind?
+  You may want to:
+    Like :heart:
+    Share :speech_balloon:
+    Contribute :pencil:
+</tree>
+</variable>
+</include>
+</box>
+
+You can use Tree in combination with inline Markdown or inline HTML elements.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+<tree>
+Formatting is supported!
+  textbook/
+    `index.md`
+  **index.md**
+    ~~index.md~~
+    %%dimmed%%
+    Super^script^
+    ****Super Bold****
+    !!Underline!!
+    !!Underline with {text=danger}!!{.text-danger}
+    <span class="text-danger">RED</span>
+  __reading.md__
+  ++site.json++
+  ==hello==
+  :construction:
+  :fas-file-code:
+  :)
+</tree>
+<tooltip id="tt:trigger_id" content="This tooltip triggered by a trigger"></tooltip>
```

Comment:
Is there supposed to be a tooltip example? Looks like it is missing the content and not showing up in the code output.